### PR TITLE
Add a script version for kube-in-docker

### PR DESCRIPTION
**Pull request by resouer**
We used a script to run this and it works very well, so we'd like to propose it here for anyone interested or lazy as we are :)

### CODE REVIEW

**Comment by thockin** [comment]
I'm assigning to Brendan.  My inclination is to say we should host the scripts and docs in-tree rather than a link to them.

**Comment by resouer** [comment]
@thockin We'd like to, if @brendandburns   thought in-tree is OK.

**Comment by brendandburns** [comment]
Yes, we'd really rather host it.  That way we can keep it up to date.

If you refactor this PR to add the script, that would be great.

Best
--brendan

**Comment by resouer** [comment]
I'd like to, will update it here soon.

File: docs/getting-started-guides/docker-multinode/master.sh [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+}
+
+detect_lsb() {
+	case "$(uname -m)" in
+	*64)
+		;;
+	*)
+		cat >&2 <<-'EOF'
+		Error: you are not using a 64bit platform.
+		We currently only supports 64bit platforms.
```
Comment (brendandburns) [inline docs/getting-started-guides/docker-multinode/master.sh]: "We currently only support 64-bit platforms"

File: docs/getting-started-guides/docker-multinode/master.sh [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+	command -v "$@" > /dev/null 2>&1
+}
+
+detect_lsb() {
+	case "$(uname -m)" in
+	*64)
+		;;
+	*)
+		cat >&2 <<-'EOF'
+		Error: you are not using a 64bit platform.
```
Comment (brendandburns) [inline docs/getting-started-guides/docker-multinode/master.sh]: 64-bit

File: docs/getting-started-guides/docker-multinode/master.sh [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+		We currently only supports 64bit platforms.
+		EOF
+		exit 1
+		;;
+	esac
+
+	if command_exists lsb_release; then
+		lsb_dist="$(lsb_release -si)"
+	fi
+	if [ -z "$lsb_dist" ] && [ -r /etc/lsb-release ]; then
```
Comment (brendandburns) [inline docs/getting-started-guides/docker-multinode/master.sh]: curly braces around all variable dereferences e.g. 
```sh
${lsb_dist}
```

File: docs/getting-started-guides/docker-multinode/master.sh [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+
+if [ "$(id -u)" != "0" ]; then
+  echo >&2 "Please run as root"
+  exit 1
+fi
+
+#TODO check is docker is running
+
+command_exists() {
+	command -v "$@" > /dev/null 2>&1
```
Comment (brendandburns) [inline docs/getting-started-guides/docker-multinode/master.sh]: fix indent (here and elsewhere)

File: docs/getting-started-guides/docker-multinode/master.sh [hunk 5]
```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
```
Comment (brendandburns) [inline docs/getting-started-guides/docker-multinode/master.sh]: Please add the boilerplate copyright header.

File: docs/getting-started-guides/docker-multinode/worker.sh [hunk 6]
```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
```
Comment (brendandburns) [inline docs/getting-started-guides/docker-multinode/worker.sh]: copyright header.

File: docs/getting-started-guides/docker-multinode/worker.sh [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+
+set -e
+
+if (! ps -ef|grep "/usr/bin/docker" |grep -v 'grep' &> /dev/null  )
```
Comment (brendandburns) [inline docs/getting-started-guides/docker-multinode/worker.sh]: space after '|'

File: docs/getting-started-guides/docker-multinode/master.sh [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+
+set -e
+
+if (! ps -ef|grep "/usr/bin/docker" |grep -v 'grep' &> /dev/null  )
+then
```
Comment (brendandburns) [inline docs/getting-started-guides/docker-multinode/master.sh]: if and then on the same line (for consistency)

**Comment by resouer** [comment]
@brendandburns Thanks for your time, I just update the scripts

**Comment by brendandburns** [comment]
@resouer The scripts still need the boilerplate copyright header

**Comment by resouer** [comment]
@brendandburns I think I've already added copyright header (copied from: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/cluster/gce/upgrade.sh). Tell me if there's mistake.

**Comment by brendandburns** [comment]
LGTM.  Thanks!

**Comment by lavalamp** [comment]
Trying to get shippable to rerun.

**Comment by lavalamp** [comment]
Shippable is a known flake.

**Comment by lavalamp** [comment]
@k8s-bot test this please

**Comment by resouer** [comment]
@lavalamp  I rebased it, seems make Shippable comfort. :)

**Comment by lavalamp** [comment]
still LGTM

**Pull request merged**